seds/mongodb-sed/values.yaml

```diff
@@ -0,0 +1,8 @@
+mongodb:
+  sed:
+    hostname: mymongo-mongodb
+    port: 27017
+    username: root
+    password: PCfT5MLN4Q
```

Comment:
change this password to a generic value like mypassword

Reply:
okay